tests/test_error_counter.py

```diff
@@ -0,0 +1,12 @@
+import io
+
+from alogamous import analyzer, error_counter_analyzer
+
```

Comment:
Can we add a test for if there are no input lines? And if there are no error lines?